api/environments.md

```diff
@@ -23,6 +27,8 @@ Maintenance can be planned on working days from 19:00 to 23:00 UTC \(20:00 to 00
 
 ## Acceptance test environment
 
+TODO: this points to the api listing. Should this point to the gitbook instead?
```

Comment:
It can only point to a listing in Gitbook. The current listings will be internal behind VPN.